internal/server/v2/facet/contained_in.go

```diff
@@ -103,6 +103,8 @@ func ContainedInFacet(
 						continue
 					}
 					seenFacets[facetID] = struct{}{}
+					// TODO: Add additional ObsCount, EarliestDate, LatestDate information.
```

Comment:
Can you file a bug for this? @n-h-diaz can you help add this in Flume?

Reply:
filed bug: https://buganizer.corp.google.com/issues/311225915